projects/core/src/product/store/effects/product-reviews.effect.ts

```diff
@@ -66,6 +66,6 @@ export class ProductReviewsEffects {
 
   constructor(
     private actions$: Actions,
-    private occProductReviewsService: ProductReviewsLoaderService
+    private productReviewsConnector: ProductReviewsConnector
```

Comment:
The concrete connector implementation is meant to be optional. By default we provide it in the OCC module, going forward, this module is optional. To that reason, should we make the connector optional, or let the whole thing intentionally break when there's no concrete implementation available in the system? I guess i've answered myself already ;) but appreciate your input.

Reply:
Yes, a concrete implementation of an adapter could be made optional in a connector, there could be even some use cases for this. Currently, it will break at runtime.